openapiart/bundler.py

```diff
@@ -366,20 +366,20 @@ def _validate_response_uid(self):
                     )
 
     def _validate_required_responses(self):
-        """Ensure all paths include a 400 and 500 response.
+        """Ensure all paths include a 200 and default response.
 
         Print every path that does not include a 400 or 500 response.
 
         Returns
         -------
-        Exception: one or more paths is missing a 400 or 500 response
-        None: all paths have a 400 and 500 response
+        Exception: one or more paths is missing a 200 or default response
+        None: all paths have a 200 and default response
         """
         responses = self._get_parser("$..paths..responses").find(self._content)
-        required_error_codes = ["400", "500"]
+        required_responses = ["200", "default"]
         missing_paths = ""
         for response in responses:
-            missing = set(required_error_codes).difference(
+            missing = set(required_responses).difference(
                 set(response.value.keys())
             )
             if len(missing):
```

Comment:
i know it's existing code, but check should be `> 0`. Moreover, we don't need to print the error, since we're raising exception already.

Reply:
done